scripts/install/pcl_install.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e  # exit on first error
+
+install_dependencies() {
+    sudo apt-get -y install -qq \
+        libboost-all-dev \
+        libflann-dev \
+        libvtk6-dev \
+        libeigen3-dev \
+        libusb-1.0-0-dev
+}
+
+echo "Installing PCL ..."
+
+install_dependencies
+
+sudo add-apt-repository -y ppa:v-launchpad-jochen-sprickerhof-de/pcl
```

Comment:
Do we have to use a PPA version? should we be compiling from source instead or using the one in ROS Kinetic?